misc/python/materialize/data_ingest/executor.py

```diff
@@ -80,6 +80,29 @@ def execute(self, cur: pg8000.Cursor, query: str) -> None:
             print(f"Query failed: {query}")
             raise
 
+    def execute_with_retry_on_error(
+        self,
+        cur: pg8000.Cursor,
+        query: str,
+        max_tries: int,
+        wait_time_in_sec: int,
```

Comment:
I suggest having defaults for max_tries and wait_time_in_sec here

Reply:
✅